lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -81,12 +95,33 @@ def active_step_by_step
         @active_step_navs.first
       end
 
+      def secondary_step_by_steps
```

Comment:
Is this method needed? I think you can just reuse `secondary_to_step_navs`.

Reply:
this has been removed